libs/ktem/ktem/reasoning/simple.py

```diff
@@ -253,6 +255,9 @@ def invoke(
             messages = []
             if self.system_prompt:
                 messages.append(SystemMessage(content=self.system_prompt))
+            for human, ai in history[-5:]:
```

Comment:
please make this an argument, a number of recent messages and/or the number of recent tokens